Tests/Rewriters/IgnoreArgumentsRewriterTests.cs

```diff
@@ -0,0 +1,248 @@
+﻿//  Copyright (c) rubicon IT GmbH
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editing;
+using NUnit.Framework;
+using RhinoMocksToMoqRewriter.Core.Rewriters;
+
+namespace RhinoMocksToMoqRewriter.Tests.Rewriters
+{
+  [TestFixture]
+  public class IgnoreArgumentsRewriterTests
+  {
+    [TestFixture]
+    public class MethodRewriterTests
+    {
+      private IgnoreArgumentsRewriter _rewriter;
+
+      private readonly Context _context =
+          new Context
+          {
+              //language=C#
+              UsingContext =
+                  @"
+using Moq;
+using MockRepository = Rhino.Mocks.MockRepository;
+#nullable enable",
+              //language=C#
+              InterfaceContext = @"
+int DoSomething (Func<int> i);
+int DoSomething (int i);
+int DoSomething (Func<int> i, Func<bool> b);
+int DoSomething (Func<int> i, Func<bool> b, string s);
+T Do<T>(Func<T> func);
+int Do (Func<int, string> func);
+int DoSomethingDifferent(Func<Func<int>, string> func);
+int Do();
+int DoSomethingInt (int? a);
+int DoSomethingIntInt (int? a, int? b);
+int DoSomethingFuncIntBool (Func<int?, bool?> f);",
```

Comment:
I think the method names should follow the same scheme if possible. They should all be named DoSomething if possible. This makes writing tests easier.